packages/mdc-layout-grid/_mixins.scss

```diff
@@ -48,6 +48,9 @@
 
   @supports (display: grid) {
     width: auto;
+    max-width: 100%;
+    max-height: 100%;
+    overflow: hidden;
```

Comment:
okay, so, backing _way_ up here:

What's the reason for attaching `mdc-layout-grid__cell` to the `<img>` element within an additional `mdc-layout-grid__cell` element? Cells aren't supposed to be nested within other cells; that behavior is actually undefined.